engine/src/configuration/game_config.cpp

```diff
@@ -1,4 +1,13 @@
-#include "game_config.h"
+#include "configuration/game_config.h"
+
+// Global Variable Definitions
+WeaponsConfig weapons_config;
```

Comment:
We should have a single config object for everything, and weapons config can be a sub-object of that object.

Reply:
I do agree on this point.

Reply:
Done.